packages/lib/gpc/src/gpcChecks.ts

```diff
@@ -669,17 +674,6 @@ export function checkVerifyClaimsForConfig(
   boundConfig: GPCBoundConfig,
   revealedClaims: GPCRevealedClaims
 ): void {
-  // Every configured POD should be revealed, with at least signing key.
```

Comment:
This would imply that every configured POD should be revealed. Should we make that a requirement? This would rule out cases where neither signing key nor entries are revealed but only constrained some other way (e.g. via `equalsEntry` or `isMemberOf`).

Reply:
You're right, ignore this comment.  With some churn on the design here I got confused over which parts of the implementation changes were driven by the inclusion of virtual entries in the entries list, and which were simply driven by the ability to reveal nothing about a POD.